features/uint8array-base64-hex.yml

```
@@ -0,0 +1,12 @@
+name: Uint8Array base64 and hex conversion
+description: The `Uint8Array` class provides the `Uint8Array.fromBase64()`, `toBase64()`, and `setFromBase64()` methods for conversion to/from base64 strings, as well as the `Uint8Array.fromHex()`, `toHex()`, and `setFromHex()` methods for conversion to/from hex strings.
```

Optional, but I think this would bring us closer to our conventions in other similar descriptions:

```suggestion
description: The `Uint8Array` object methods `fromBase64()`, `toBase64()`, and `setFromBase64()` convert to and from base64 strings. The `fromHex()`, `toHex()`, and `setFromHex()` methods convert to and from hex strings.
```

Alternatively, we could split this in two (one feature for hex and one for base64) though they're very likely to differ only in type, not support stories.